### DISCUSSION TITLE
Endless Campaigns

### DISCUSSION THREAD

**Discussion by catalin-duta** [Keriew/augustus#509] [Ideas]
First of all, hello to everyone and thank you for the great work on this project!

tl;dr  End-of-campaign option to "Continue governing forever"

After spending many years building and rebuilding a city, I would love to have the option to keep perfecting it and expanding it. As it is, once the victory conditions are met, I only get at most 5 years to keep working on the city.

Yes, the Assignment Editor and City Construction Kit exist, but these require starting a city/campaign from scratch and I get the impression that some of the maps are different - for example Lugdunum in the campaign has an elevated and hard to reach farmland, overall very rugged terrain, while the Lugdunum map in CCK is very open and bearing little resemblance.

I switched from Julius to Augustus while on the Lugdunum campaign, and curiously my Culture and Prosperity ratings made a huge jump - even after replacing Colloseums with Arenas, and without having ever crafted wine to reach Patrician housing. Within 2 months of loading the save game that was deliberately not close to victory conditions, I was promoted to Aedile.

Is there a possibility to add an option to "Continue governing indefinitely"? Or maybe, instead of the automatic jump to the next mission on the extra 2/5 years expiring, to be prompted again whether to move on to the next campaign or to continue governing?

**Comment by Ouaz** [comment]
> Is there a possibility to add an option to "Continue governing indefinitely"? Or maybe, instead of the automatic jump to the next mission on the extra 2/5 years expiring, to be prompted again whether to move on to the next campaign or to continue governing?

It's actually already possible in Augustus, implemented in version 2.0: :yum:
"_Allow indefinite play of campaign mission: after extending your mandate for a few years, the player will be asked again to accept the promotion or extend the regency._"

> > I switched from Julius to Augustus while on the Lugdunum campaign, and curiously my Culture and Prosperity ratings made a huge jump - even after replacing Colloseums with Arenas, and without having ever crafted wine to reach Patrician housing. Within 2 months of loading the save game that was deliberately not close to victory conditions, I was promoted to Aedile.

Regarding Culture rating, I think it's due to schools, libraries, and theaters having an upgrade (i.e. second tier) in Augustus (implemented in 3.0).

"_Caesar 3’s concept of “Culture Rating” required the player to balance the venue capacity of certain structures against the total population of the city. This inevitably led to a situation players called “culture dumps”; or excessive placement of structures like Schools, Libraries, Theaters, and Academies, often in the very last days of a mission. To provide an alternative, we have implemented structural upgrades similar to how Bathhouses, Markets, Fountains, and Senate work._ 

_Theater, School, and the Library now have a new upgrade state tied to desirability. When the sufficient desirability level is achieved, the structure will visually change - in addition, it will also increase the venue capacity of the structure for the purpose of calculating Culture rating. For example: a standard Theater covers 500 people, while an upgraded Theater covers 1000 people. To save labor and space, you should strive to place these cultural structures in areas with high desirability._"

In your Julius save, maybe you built a lot of theaters, schools, and libraries (in high desirability area), and launching the save under Augustus, they evolved to second tier, causing a big culture rating jump.

But for Prosperity, I don't see what could have increased it. Maybe very happy citizens? :thinking:

**Reply by catalin-duta** [reply]
Thank you! I did not know that it is possible. I had gone through the manual, but that seems to list major changes and seems focused on 3.0. I don't know my way around here that well yet so I hadn't come upon the 2.0 changelog.

For Culture: it would seem to be about the upgrades, to Library in my case. I don't have many Theaters and only 1 in 10 Schools upgraded compared with 9 in 10 Libraries, although they're in the same general area with around the same desirability. I guess they have different min-desirability. Removing a couple of Academies got me back to the pre-Augustus Culture level.

Regarding Prosperity: my memory of the original game is not that fresh, but I was under the impression that Prosperity would not go beyond a certain level without a specific level of housing. In Augustus (not Julius) it does seem to increase yearly if the city is stable enough (mostly happy citizens, low unemployment, some profit). That with plenty of city replanning and rebuilding breaking the balance of the above for a few months every year.

I'll keep an eye on this part and do a bit more reading on the expected outcome so I'm in the know.
